packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/multiphysics/DecapodesContext.java

```diff
@@ -0,0 +1,17 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.multiphysics;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import java.io.Serializable;
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.models.dataservice.TerariumAsset;
+
+@Data
+@Accessors(chain = true)
+@TSModel
+public class DecapodesContext extends TerariumAsset implements Serializable {
+	private ContextHeader header;
+	private Context context;
```

Comment:
Do we need to define those so perfectly?
I do not mind, but this can change at any point and will be quite difficult to support. Same thing for `configuration`.
Maybe we should add support for the `modelVersion` in the `ModelHeader`. But so close to the hackathon...

Reply:
Switching to Object/JsonNode would be trivial.